src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,48 @@
 package mate.academy;
 
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final MovieService movieService =
+            (MovieService) injector.getInstance(MovieService.class);
+    private static final MovieSessionService movieSessionService =
+            (MovieSessionService) injector.getInstance(MovieSessionService.class);
+    private static final CinemaHallService cinemaHallService =
+            (CinemaHallService) injector.getInstance(CinemaHallService.class);
 
+    public static void main(String[] args) {
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        Movie venom = new Movie("Venom");
+        venom.setDescription("Excellent action film with Tom Hardy in main role"
+                + ", good choice for look");
+        movieService.add(venom);
+        System.out.println(movieService.get(venom.getId()));
+        movieService.getAll().forEach(System.out::println);
+
+        CinemaHall yourChoice = new CinemaHall("Your choice");
```

Comment:
The `CinemaHall` constructor is being called with a string argument, which is not consistent with the constructor definition in `CinemaHall.java`. The constructor in `CinemaHall.java` takes a `String` for the description, but it seems like the intention here is to set the name or identifier of the cinema hall. Ensure that the constructor and its usage align with the intended design.